File: Python Assignments/Module-8 Advance Python Programming/5. Exception Handling/Q-4.py
# Write a Python program to demonstrate handling multiple exceptions.

def handlingException():
     # taking input
    num1 = int(input("Enter first number: "))
    num2 = int(input("Enter second number: "))

    # performing division
    result = num1 / num2

    print(f"The result of division is: {result}")

# handling multiple exceptions
try:
    handlingException()
except (ValueError, ZeroDivisionError) as e:
    print("Invalid input. Try again")
    print("Error: ",e)

File: module-3 Assignment/Conditional Logic Programs/Looping & Conditional Statements/Q4.c
#include<stdio.h>
int main(){
	int num1,num2;
	
	printf("Enter the number of table you want to print: ");
	scanf("%d",&num1);
	printf("Enter the range: ");
	scanf("%d",&num2);
	
	for(int i=1;i<=num2;i++){
		printf("\n%d x %d = %d",num1,i,num1*i);
	}
	
	return 0;
}
